templates/src/zigbee/carbon_dioxide.tpl

```diff
@@ -36,6 +36,7 @@ typedef void * zb_voidp_t;
     ZB_ZCL_ATTR_CARBON_DIOXIDE_VALUE_ID, \
     ZB_ZCL_ATTR_TYPE_SINGLE, \
     ZB_ZCL_ATTR_ACCESS_READ_ONLY | ZB_ZCL_ATTR_ACCESS_REPORTING, \
+    (ZB_UINT16_MAX), \
```

Comment:
This is a breaking change, but it should be okay. Will mention this in release notes.

Reply:
On the second thought this change is not major enough, and in configuration we have some functionality to [parse & compare versions](https://github.com/ffenix113/zigbee_home/blob/86718037b1aabed130653c805a980f4cbd59e82c/config/ncs_finder.go#L112), which could be used as a guard here as well. After this is merged, I will add the checks to make it compatible with <v2.6.0 & >= v2.6.0. :+1: